src/compiler/checker.ts

```diff
@@ -49131,8 +49131,11 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
 
     function getEffectivePropertyNameForPropertyNameNode(node: PropertyName) {
         const name = getPropertyNameForPropertyNameNode(node);
-        return name ? name :
-            isComputedPropertyName(node) && isEntityNameExpression(node.expression) ? tryGetNameFromType(getTypeOfExpression(node.expression)) : undefined;
+        return name

```

Comment:
This formatting change doesn't seem required; can you just flip this back to the way it was and just drop the condition?

Reply:
Wait, I'm confused, what formatting change?  It's just replacing a whole nested ternary with `return name;` (unless GitHub is hiding part of the diff...)

Reply:
Sorry, I commented on the first line; probably I should have selected the entire snippet but that would have been the entire diff in the file which felt egregious.